src/sdk/namespace_blob.js

```diff
@@ -459,15 +458,15 @@ class NamespaceBlob {
         const md5_hash = crypto.createHash('md5');
         const block_list = _.times(num_parts, num => {
             const part = part_by_num[num + 1];
-            md5_hash.update(part.etag);
-            return part.etag;
+            const block_id = Buffer.from(part.etag, 'hex').toString('ascii');
+            return block_id;
         });
-
+        const block_obj = { UncommittedBlocks: block_list, CommittedBlocks: [], LatestBlocks: [] };
```

Comment:
According to this: https://azure.github.io/azure-storage-node/BlobService.html
`The wrapper for block ID list contains block IDs that make up the blob.
Three kinds of list are provided, please choose one to use according to requirement.`
I would change to: { UncommittedBlocks: block_list }.
We aren't interested in sending other parameters and they are not required.
It also requests to send one list, so sending several lists (even if empty) might create a bug in the future.